src/util/dsp/LogSlider.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+namespace otto::util::dsp {
+  class LogSlider
```

Comment:
Is the whole slider concept a little weird? dont know what else to call it, but i think `proportion_of_length` is a little specific.

This is just a map `[0;1] -> [min;max]` with a logarithmic scale, right?
Maybe slider is an alright word for it, cant come up with anything better anyway